src/gobexport/exporter/config/brk.py

```diff
@@ -372,8 +378,8 @@ class AantekeningenExportConfig:
             'value': 'Aantekening Zakelijk Recht (R)'
         },
         'BRK_KOT_ID': 'rustOpKadastraalobject.[0].identificatie',
-        'KOT_KADASTRALEGEMCODE_CODE': 'rustOpKadastraalobject.[0].aangeduidDoorKadastralegemeentecode.broninfo.code',
-        'KOT_SECTIE': 'rustOpKadastraalobject.[0].aangeduidDoorKadastralesectie.bronwaarde',
+        'KOT_KADASTRALEGEMCODE_CODE': 'rustOpKadastraalobject.[0].aangeduidDoorKadastralegemeentecode.bronwaarde',
+        'KOT_SECTIE': 'rustOpKadastraalobject.[0].aangeduidDoorKadastralesectie.[0].code',
```

Comment:
Won't this become aangeduidDoorKadastralesectie.[0].code ? Without rustOpKadastraalobject?